dhcpv6/duid.go

```diff
@@ -28,6 +31,7 @@ type Duid struct {
 	LinkLayerAddr        []byte
 	EnterpriseNumber     uint32 // for DUID-EN. Ignored otherwise
 	EnterpriseIdentifier []byte // for DUID-EN. Ignored otherwise
+	Uuid                 [16]byte // for DUID-UUID. Ignored otherwise
```

Comment:
I'd rather have a `[]byte` and do the appropriate checks elsewhere in the code. My experience with arrays in this type of use is that it's more painful to cast/convert things around than just dealing with making sure that we have the right number of bytes

Reply:
Ok, I'll update it.